activestorage/lib/active_storage/service/azure_storage_service.rb

```diff
@@ -116,6 +116,22 @@ def url_for_direct_upload(key, expires_in:, content_type:, content_length:, chec
       end
     end
 
+    def public_url(key, filename:, disposition:, content_type:)
```

Comment:
This still technically generates a signed URL. I'll have to read up on the azure gem to see how to do this properly.